### PR TITLE
feature/18-edit&DeleteProductsWithAdmin

### PR DESCRIPTION
**Pull request by Tiago-SR**

### CODE REVIEW

File: src/app/components/product-create/product-create.component.ts [hunk 1]
```diff
@@ -8,32 +9,59 @@ import { fileValidator } from "../../validators/file-validator";
...
+    private fb: FormBuilder,
+    private productService: ProductService,
+    private route: ActivatedRoute,
+    private router: Router
+  ) {
+    this.productForm = this.fb.group({
+      name: ['', Validators.required],
+      price: ['', [Validators.required, Validators.pattern('^[0-9]*$')]],
+      description: ['', Validators.required],
+      image: ['', fileValidator(['image/jpeg', 'image/png'], 2 * 1024 * 1024)]
```
Comment (MatiasBelsterli) [inline src/app/components/product-create/product-create.component.ts:31]: are '.jpg' files included under the 'jpeg' extension?

File: src/app/components/product-create/product-create.component.ts [hunk 2]
```diff
@@ -8,32 +9,59 @@ import { fileValidator } from "../../validators/file-validator";
...
     });
   }
 
+  loadProductData() {
+    this.productService.getProductById(Number(this.productId)).subscribe(product => {
+      this.productForm.patchValue({
+        name: product.name,
+        price: product.price,
+        description: product.description,
+        // No cargamos la imagen, ya que no podemos previsualizarla
```
Comment (MatiasBelsterli) [inline src/app/components/product-create/product-create.component.ts:55]: sorry what?

File: src/app/components/product-create/product-create.component.ts (hunk 1 above)
Comment (Tiago-SR) [reply src/app/components/product-create/product-create.component.ts:31]: Yes, the '.jpg' are included in '.jpeg'

File: src/app/components/product-create/product-create.component.ts (hunk 2 above)
Comment (Tiago-SR) [reply src/app/components/product-create/product-create.component.ts]: I'm sorry, I don't know what I meant

**Pull request merged**